sc-locked-asset-factory/src/locked_asset.rs

```diff
@@ -48,6 +54,43 @@ pub trait LockedAssetModule: asset::AssetModule {
         );
     }
 
+    fn send_tokens(
+        &self,
+        token: &TokenIdentifier,
+        nonce: Nonce,
+        amount: &Self::BigUint,
+        destination: &Address,
+        opt_accept_funds_func: &OptionalArg<BoxedBytes>,
+    ) -> SCResult<()> {
+        let (function, gas_limit) = match opt_accept_funds_func {
+            OptionalArg::Some(accept_funds_func) => (
+                accept_funds_func.as_slice(),
+                self.transfer_exec_gas_limit().get(),
+            ),
+            OptionalArg::None => {
+                let no_func: &[u8] = &[];
+                (no_func, 0u64)
+            }
+        };
+
+        let result = self.send().direct_esdt_nft_execute(
```

Comment:
you should not care about the result when sending the tokens

Reply:
Fixed